src/Forms/Shared/Samples/Symbology/CustomDictionaryStyle/CustomDictionaryStyle.xaml.cs

```diff
@@ -30,45 +32,83 @@ public partial class CustomDictionaryStyle : ContentPage
         // Uri for the restaurants feature service.
         private readonly Uri _restaurantUri = new Uri("https://services2.arcgis.com/ZQgQTuoyBrtmoGdP/arcgis/rest/services/Redlands_Restaurants/FeatureServer/0");
 
+        // Hold a reference to the custom dictionary styles for use in the event handlers. 
+        private DictionarySymbolStyle _localRestaurantStyle;
+        private DictionarySymbolStyle _restaurantStyleFromPortal;
```

Comment:
I wonder if it would set a better performance example to create and store the renderer, rather than re-creating it from the style each time?